### PR TITLE
fix(sync-actions): generate PriceCustom actions before changePrice

### PR DESCRIPTION
**Pull request by MarkusAzer**
#### Summary

Fix for issue [1561](https://github.com/commercetools/nodejs/issues/1561)

#### Description

Generating `PriceCustom` actions before `changePrice` to fix actions overloading each other.
The ordering of the actions tested via Impex

### CODE REVIEW

**Comment by MarkusAzer** [comment]
> I guess here ordering update was the simplest solution and looks good to me, I have not tried it but do we know it will pass an integration test, did you have a chance to run some tests against platform?
> 
> Also even it works, another concern for me here is if we change prices with the same custom fields, do we actually need to run another action here before ? Especially performance-wise 🤔

yep regarding tests with platform, I simulated it with Impex and so it works as expected.

regarding actions, we wanted to cover all action which platform accepts.

there is solution regarding ignoring some actions while generating it, which is basically just pickup actions which you need with specify actionGroups.

**Comment by MarkusAzer** [comment]
> thanks for the fix @MarkusAzer 🎉

Big Thanks @ahmetoz :) for reporting the issue 🥇

**Comment by ahmetoz** [comment]
@MarkusAzer what is the plan for release?

**Comment by MarkusAzer** [comment]
> @MarkusAzer what is the plan for release?

I'm just waiting for the team to review it and then, it should be realised, i believe mostly today

File: packages/sync-actions/test/product-sync-prices.spec.js [hunk 1]
```diff
@@ -169,6 +169,101 @@ describe('Actions', () => {
     expect(actions).toEqual([])
   })
 
+  test('should generate PriceCustom actions before changePrice action', () => {
```
Comment (katmatt) [inline packages/sync-actions/test/product-sync-prices.spec.js:172]: Perfect and thanks for adding this test 👍 This way we can be sure that further changes won't break it 😎

Comment (daern91) [inline packages/sync-actions/test/product-sync-prices.spec.js:172]: Super!

**Pull request merged**